src/main/java/com/scalar/db/storage/multistorage/MultiStorage.java

```diff
@@ -29,43 +29,68 @@
  * A storage implementation with multi-storage for {@link DistributedStorage}.
  *
  * <p>This storage implementation holds multiple storage instances. It chooses a storage instance on
- * the basis of the specified configuration and a given operation.
+ * the basis of the specified configuration and a given operation. If conflicting mapping between a
+ * table mapping and a namespace mapping, it prefers the table mapping because table mappings are
```

Comment:
```suggestion
 * the basis of the specified configuration and a given operation. If there is a conflict between a
 * table mapping and a namespace mapping, it prefers the table mapping because table mappings are
```

Reply:
We might need to reformat after the change.